src/strife/d_main.c

```diff
@@ -234,6 +234,7 @@ void D_Display (void)
     if (gamestate != wipegamestate)
     {
         screenwipe = true; // [crispy]
+        fade_safe_tics = CROSSFADETICS; // [crispy] arm fail-safe crossfade counter
```

Comment:
If you could move this line to `f_wipe.c`, then the whole `fade_safe_tics` dance could remain static in that file.

Reply:
Done in https://github.com/fabiangreffrath/crispy-doom/pull/1183/commits/93f36f377cd365963256412b44f22eb2708951dc. Turns out, it wasn't a "tics", it's "crossfading iterations".